### ISSUE TITLE
joysetup.sh fails on lxcfs permission errors

### ISSUE THREAD

**Issue by Smithx10** [joyent/linux-live#16]
```diff
diff --git a/scripts/joysetup.sh b/scripts/joysetup.sh
index fda7ffbd..663affd0 100755
--- a/scripts/joysetup.sh
+++ b/scripts/joysetup.sh
@@ -700,7 +700,7 @@ setup_datasets()
         # shellcheck disable=2064
         trap "cp /tmp/joysetup.$$ /var/log/joysetup.log" EXIT
 
-        if ( ! find . -print | TMPDIR=/tmp cpio -pdm "/${VARDS}" ); then
+        if ( ! find . -not -path '*/lib/lxcfs*' -print | TMPDIR=/tmp cpio -pdm "/${VARDS}" ); then
             fatal "failed to initialize the var directory"
         fi
 ```

**Comment by danmcd** [comment]
Testing notes?

**Comment by danmcd** [comment]
According to joyent/sdc-headnode#53 this was tested by the filer.

**Comment by bahamat** [comment]
Fixed in joyent/sdc-headnode@1a619bc
